static/sidebar/js/modal/index.js

```diff
@@ -54,14 +54,19 @@
       prodNode.href = productLink;
       prodNode.innerHTML = '<div class="tvp-product-image" '+productImgStyle+'><div/>';
       thumbsFrag.appendChild(prodNode);
+      
```

Comment:
Protect your code, if `title` or `price` etc is not defined code will brake
We are also using product.title few times, lets try to tackle both like:

`var trimmedTitle = productTitle.length > 50 ? productTitle.substring(0, 50 - 3) + "..." : productTitle;`